src/remote/watcher.js

```diff
@@ -56,6 +71,9 @@ export default class RemoteWatcher {
       await this.pouch.setRemoteSeqAsync(changes.last_seq)
       log.lineBreak()
     } catch (err) {
+      if (err.message === 'Client has been revoked') {
```

Comment:
I've tried to subclass Error, but babel has some caveats on subclassing Error and it didn't work.